src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -101,21 +97,6 @@ export const PaymentRoute: FC<PaymentRouteProps> = props => {
 
   const artworkVersion = extractNodes(order.lineItems)[0]?.artworkVersion
 
-  useEffect(() => {
-    if (!bankAccountSelection && selectedPaymentMethod) {
-      const bankAccountsArray =
-        selectedPaymentMethod !== "SEPA_DEBIT"
-          ? extractNodes(me.bankAccounts)
-          : []
-
-      setBankAccountSelection(
-        getInitialBankAccountSelection(order, bankAccountsArray)
-      )
-    }
```

Comment:
This is more concise than what I am proposing, but it makes it harder to read this hook an understand what it is doing.